Cargo.toml

```diff
@@ -159,6 +143,44 @@ pallet-utility = { git = "https://github.com/paritytech/substrate", branch = "po
+fp-dynamic-fee = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+fp-ethereum = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+fp-evm = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+fp-rpc = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+fp-self-contained = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+fp-storage = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+
+# Frontier FRAME
+pallet-base-fee = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-dynamic-fee = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-ethereum = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-chain-id = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-blake2 = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-bn128 = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-curve25519 = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-dispatch = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-ed25519 = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-modexp = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-sha3fips = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-precompile-simple = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+pallet-evm-test-vector-support = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39" }
+pallet-hotfix-sufficients = { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
+
+# Parachain dependencies
+substrate-fixed = { git = "https://github.com/encointer/substrate-fixed", default-features = false }
```

Comment:
cc @1xstj does we still need this?

Reply:
Yes we do. It's needed for some dep I recall hitting an error without it.